src/library/Uno.Toolkit.WinUI.Markup/Theme/TabBar.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+
+namespace Uno.Toolkit.Markup;
+
+public static partial class Theme
+{
+	public static class TabBar
+	{
+		public static class Resources
+		{
+			public static class Default
+			{
+				public static class Item
+				{
+					public static class Background
```

Comment:
We should be creating just a `TabBarItem` class here that is outside of the `TabBar` class